src/lib.rs

```diff
@@ -210,6 +211,36 @@ impl LogLevel for InfoLevel {
     }
 }
 
+/// Default to [`log::Level::Debug`]
+#[derive(Copy, Clone, Debug, Default)]
+pub struct DebugLevel;
+
+impl LogLevel for DebugLevel {
+    fn default() -> Option<log::Level> {
+        Some(log::Level::Debug)
+    }
+}
+
+/// Default to [`log::Level::Trace`]
+#[derive(Copy, Clone, Debug, Default)]
+pub struct TraceLevel;
+
+impl LogLevel for TraceLevel {
+    fn default() -> Option<log::Level> {
+        Some(log::Level::Trace)
+    }
+}
+
```

Comment:
Please remove these from the PR as they need their own issue / discussion, see #80

Reply:
removed